kafka/rdkafka/consumer.py

```diff
@@ -20,19 +21,27 @@
 
 class SimpleConsumer(base.BaseSimpleConsumer):
 
-    def __init__(self, client, topic, partitions=None):
-        if isinstance(topic, basestring):
-            topic = client[topic]
-        self._topic = topic
-        self._partitions = partitions or self.topic.partitions # ie all
+    def __init__(self, *args, **kwargs):
```

Comment:
Thanks, will change it. I was mostly worried about duplicating the default arguments (I think we'd notice if the argspec got out of sync between the py and rd_kafka implementations, through the test suite, but we might miss it when default arguments would get out of sync?).

This inspect.getcallargs pattern was also part of pullreq #133 that you merged - I'll create a patch reverting that part.